scripts/pageLngSupport.js

```diff
@@ -0,0 +1,19 @@
+function changeLang() {
+    var x="";
+    x = document.getElementById("languageSelection").value;
+    
+    if(x=="ga")
+    {
+        document.getElementById("title").innerHTML = "Seiceálaí Litrithe";
+        document.getElementById("webLang").innerHTML = "Roghnaigh teanga leathanach gréasáin: ";
+        document.getElementById("text").innerHTML = "Iontráil do théacs thíos: ";
+        document.getElementById("textLang").innerHTML = "Roghnaigh teanga téacs: ";
+    }
+    else       
+    {
+        document.getElementById("title").innerHTML = "Spell Checker";
+        document.getElementById("webLang").innerHTML = "Select webpage language:  ";
+        document.getElementById("text").innerHTML = "Enter your text below: ";
+        document.getElementById("textLang").innerHTML = "Select text language: ";
+    }
+}
```

Comment:
We might be able to create an object with this information for the final version

Reply:
ok